lib/class.Linfo.php

```diff
@@ -42,7 +42,7 @@ class Linfo {
 		$version = '',
 		$time_start = 0;
 
-	public function __construct() {
```

Comment:
Remove "array" type hinting and use the old array() syntax. We're targetting old php 5.2 which is shipped with rhel6.